tutorials/math/exampleFunction.py

```diff
@@ -0,0 +1,95 @@
+## \file
+## \ingroup tutorial_math
+## \notebook
+## Example of using Python functions and input to numerical algorithm
+## using the ROOT Functor class 
+##
+## \authors Lorenzo Moneta
+
+import ROOT
+import math
+
+## example 1D function
+
+def f(x):
+   return x*x -1
+
+func = ROOT.Math.Functor1D(f)
+
+#example of using the integral
+
+print("Use Functor1D for wrapping one-dimensional function and compute integral of f(x) = x^2-1")
+
+ig = ROOT.Math.Integrator()
+ig.SetFunction(func)
+
+value = ig.Integral(0, 3)
+print("integral-1D value = ", value)
+expValue = 6
+if (not ROOT.TMath.AreEqualRel(value, expValue, 1.E-15)) :
+   print("Error computing integral - computed value - different than expected, diff = ", value - expValue)
+   
+# example multi-dim function
+
+print("\n\nUse Functor for wrapping a multi-dimensional function, the Rosenbrock Function r(x,y) and find its minimum")
+
+def RosenbrockFunction(xx):
+  x = xx[0]
+  y = xx[1]
+  tmp1 = y-x*x
+  tmp2 = 1-x
+  return 100*tmp1*tmp1+tmp2*tmp2
+
+
+func2D = ROOT.Math.Functor(RosenbrockFunction,2)
+
+### minimize multi-dim function using fitter class
+
+fitter = ROOT.Fit.Fitter()
+initialParams = ROOT.std.vector('double')(2)
```

Comment:
To be more pythonic we can use here an `array.array` or a numpy array. So:
```python
import array
initialParams = array.array('d', [0.,0.])
```
or
```python
import numpy as np
initialParams = np.array([0.,0.], dtype='d')
```

Reply:
Thank you for the suggestion. 
I agree using numpy or array is better. 
I have updated the tutorial